Sources/FunctionalCss/GridHelpersUpgrade.swift

```diff
@@ -0,0 +1,53 @@
+import Css
+import HtmlUpgrade
+
+// TODO: extract to grid helpers in design systems?
+public func _gridRow(_ attribs: [Attribute<Tag.Div>], _ content: [Node]) -> Node {
```

Comment:
we need to duplicate any helper functions we have on `Node`, but I don't think this will be so bad (luckily we dont have to touch CSS). I am adding underscores to the duplicated functions to help with type checking times.